imports/plugins/core/ui-tagnav/client/components/tagGroupBody.js

```diff
@@ -0,0 +1,173 @@
+import debounce from "lodash/debounce";
+import update from "react/lib/update";
+import React, { Component, PropTypes } from "react";
```

Comment:
Correct me if I'm wrong, but I'm guessing @kieckhafer means line 3 should come before line 2?